exodus/exodus.py

```diff
@@ -210,7 +210,13 @@ def find(self, name):
             uri.replace('about.rdf', '')
             for uri in self.root.xpath('mods:subject/mods:geographic/@valueURI', namespaces=self.namespaces)
         ]
+        lc_uris = [
+            uri
+            for uri in self.root.xpath('mods:subject[mods:geographic]@valueURI', namespaces=self.namespaces)
```

Comment:
Change this:

```suggestion
           for uri in self.root.xpath('mods:subject[mods:geographic]/@valueURI', namespaces=self.namespaces)
```

Reply:
> Change this:

Yes, this one I almost changed myself before opening the PR yesterday